kustomize/base/addons/storage/storage.yaml

```diff
@@ -42,11 +42,11 @@ metadata:
 spec:
   externalName: minio
   ports:
-  - port: 9000
+    - port: 9000
   selector:
     app: minio
 ---
-apiVersion: extensions/v1beta1
+apiVersion: networking.k8s.io/v1
```

Comment:
thank you for fixing this. this is related to issue https://github.com/flyteorg/flyte/issues/685